setup.py

```diff
@@ -0,0 +1,12 @@
+from setuptools import setup, find_packages
+
+
+setup(
+    name="xplaneconnect",
+    version="1.0.0",
+    python_requires=">=3",
+    packages=find_packages("Python3/src"),
+    package_dir={"": "Python3/src"},
+    description="XPlaneConnect (XPC) facilitates communication to and from the XPCPlugin.",
+    url="https://github.com/nasa/XPlaneConnect/",
```

Comment:
```suggestion
    url="https://github.com/nasa/XPlaneConnect/",
    project_urls={  # Optional
        'Bug Reports': 'https://github.com/nasa/xplaneconnect/issues',
        'Organization': 'https://www.nasa.gov/content/diagnostics-prognostics',
        'Source': 'https://github.com/nasa/xplaneconnect',
        'Documentation': 'https://github.com/nasa/XPlaneConnect/wiki'
    }
```
Let's add a few more links here